src/main/java/com/mojang/brigadier/exceptions/CommandSyntaxException.java

```diff
@@ -16,15 +16,15 @@ public class CommandSyntaxException extends Exception {
     private final int cursor;
 
     public CommandSyntaxException(final CommandExceptionType type, final Message message) {
-        super(message.getString(), null, ENABLE_COMMAND_STACK_TRACES, ENABLE_COMMAND_STACK_TRACES);
+        super(null, null, ENABLE_COMMAND_STACK_TRACES, ENABLE_COMMAND_STACK_TRACES);
```

Comment:
For the 2nd null, could you delete it because its a duplicate? Same for line 27?

Reply:
The second `null` should not be a duplicate. Since `Exception` does not have such a constructor, if you delete the second `null`, it will result in an error.